olp-cpp-sdk-core/src/cache/DiskCache.h

```diff
@@ -148,7 +149,8 @@ class DiskCache {
   std::string disk_cache_path_;
   std::unique_ptr<leveldb::DB> database_;
   std::unique_ptr<const leveldb::FilterPolicy> filter_policy_;
-  std::unique_ptr<DiskCacheSizeLimitEnv> environment_;
+  std::unique_ptr<DiskCacheSizeLimitEnv> disk_cache_size_limit_environment_;
```

Comment:
If it is not possible to use `DiskCacheSizeLimitEnv` as base class for `ReadOnlyEnv` then please keep the analogy in sync and name this either `default_environment` or `read_write_environment`.

Reply:
Done